Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -43,33 +65,46 @@ class BRBViewController: UIViewController {
             make.edges.equalToSuperview()
         }
         loginViewController.didMove(toParentViewController: self)
+        
+//        if let account = accountManager.getCachedAccount() {
+//            setState(.account(account))
+//            activityIndicator.startAnimating()
+//            loggedIn = true
+//        }
     }
     
     @objc private func aboutButtonPressed(_ sender: UIBarButtonItem) {
         let aboutVC = AboutTableViewController()
         aboutVC.delegate = self
-        
-        if case .finished = connectionHandler.stage {
+
+        if case .finished = accountManager.stage {
+            aboutVC.logoutEnabled = true
+        } else if loggedIn {
             aboutVC.logoutEnabled = true
         } else {
             aboutVC.logoutEnabled = false
         }
         
         navigationController?.pushViewController(aboutVC, animated: true)
+        navigationItem.title = "Back"
     }
     
     private func setState(_ newState: State) {
         switch (state, newState) {
         case (.login, .account(let account)):
             let accountViewController = BRBAccountViewController(account: account)
             self.accountViewController = accountViewController
-            
+            self.accountViewController?.delegate = self
             addChildViewController(accountViewController)
             view.insertSubview(accountViewController.view, at: 0)
             accountViewController.view.snp.makeConstraints { make in
                 make.edges.equalToSuperview()
             }
             accountViewController.didMove(toParentViewController: self)
+
+            if let credentials = accountManager.getCredentials() {
+                navigationItem.title = "Hello, \(credentials.netid)"
```

Comment:
Feature request: can we make this be their first name?

Reply:
Not currently and adding that feature would take a lot of work. But we can remove the numbers at the end of the user's name so that it looks cleaner and more personal by just showing their initials

Reply:
That's an interesting idea. Maybe raise it with design?

Reply:
Yeah I did, we are just going to set it to "Account Info" or something like that

Reply:
The only issue I see with it is that since the user does not have to log in each time anymore, how do we go about telling them that they are looking at their account?

Reply:
Why would we have to tell them they're looking at their account? Isn't that obvious? It's not like people are out here sharing phones.

Reply:
I think it is an important confirmation to have

Reply:
Doesn’t saying “Hello, bpd282” count as confirmation? Also, I would actually include the numbers. It might be confusing to a user to not see them given that they are so accustomed to it.

Reply:
Yes, it would, but design said it looked weird and said to make it "Account Info". In that case I am thinking we should add a section title that shows the NetID or something like that